site/index.html

```diff
@@ -63,7 +74,13 @@ <h2>Try it!</h2>
       </div>
    </div>
 </div>
+
 <br>
+<br class="hidden-sm">
+<br class="hidden-sm">
+<br class="hidden-xs hidden-sm">
+<br class="hidden-xs hidden-sm">
+
```

Comment:
Are these changes necessary?

Reply:
These are the final `<br>`s I added for the purpose of centering the content.